### PR TITLE
[MI100][MI200]Kernel Cache Update

### PR DESCRIPTION
**Pull request by cderb**
Sync with Tuna's last full fdb generation. Overlay MLIR and Fambench tunings. 
Include exported kdb

### CODE REVIEW

**Review by junliume — APPROVED** [review]

File: include/miopen/config.h.in [hunk 1]
```diff
@@ -152,4 +152,6 @@
 #include <hip/hip_version.h>
 #endif
 
+#define MIOPEN_GOLDEN_DB_VERSION 5
```
Comment (atamazov) [inline include/miopen/config.h.in:155]: @cderb Hi Chris, AFAICS this macro is not used in the library. Can you please shed some light on what it is for?

**Comment by atamazov** [comment]
@cderb 
> Sync with Tuna's last full fdb generation

Hmm... Could you please provide the name of the branch (and git hash) of MIOpen that was used for full fdb generation? This might be important. Thanks.

File: include/miopen/config.h.in (hunk 1 above)
Comment (junliume) [reply include/miopen/config.h.in:155]: @cderb have the same question

**Comment by atamazov** [comment]
@cderb @JehandadKhan Like #1841, this PR contains some outdated find-db updates for MI100/MI200 related to ConvBinWinogradRxSf2x3g1 and ConvBinWinogradRxSf2x3. 

```bash
$ for f in src/kernels/*.fdb.txt; do ls -l ${f} ; grep -E ".*-[W]=.*ConvBinWinogradRxSf2x3g1," ${f} | wc -l ; done
-rw-r--r-- 1 atamazov 197121 60585584 Nov  8 18:16 src/kernels/gfx90878.HIP.fdb.txt
7662
-rw-r--r-- 1 atamazov 197121 58362704 Nov  8 18:16 src/kernels/gfx90a68.HIP.fdb.txt
12080
-rw-r--r-- 1 atamazov 197121 58418820 Nov  8 18:16 src/kernels/gfx90a6e.HIP.fdb.txt
11961
$ for f in src/kernels/*.fdb.txt; do ls -l ${f} ; grep -E ".*-[FB]=.*ConvBinWinogradRxSf2x3," ${f} | wc -l ; done
-rw-r--r-- 1 atamazov 197121 60585584 Nov  8 18:16 src/kernels/gfx90878.HIP.fdb.txt
15038
-rw-r--r-- 1 atamazov 197121 58362704 Nov  8 18:16 src/kernels/gfx90a68.HIP.fdb.txt
1
-rw-r--r-- 1 atamazov 197121 58418820 Nov  8 18:16 src/kernels/gfx90a6e.HIP.fdb.txt
758
```
More info at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1841#issuecomment-1307373707

I will provide the fix soon.

File: include/miopen/config.h.in (hunk 1 above)
Comment (cderb) [reply include/miopen/config.h.in:155]: Tuna is now managing a "golden table" for MIOpen tunings which is a set of versioned databases for MIOpen. This define refers to the version of that table currently in MIOpen.

**Comment by cderb** [comment]
> Hmm... Could you please provide the name of the branch (and git hash) of MIOpen that was used for full fdb generation? This might be important. Thanks.

the last full fdb tuning was performed with:
mi200
miopen: develop 6fd47454b
rocm: 5.3

mi100
miopen: develop bdf6d6a16
rocm: 5.3

**Pull request merged**